app/controllers/carl_people_controller.rb

```diff
@@ -0,0 +1,47 @@
+class CarlPeopleController < ApplicationController
+  #----------------- SHOW A PERSON --------------------
+  def show
+    @person = Person.find(params[:id])
+    @tasks_count = @person.tasks.count
+    @tasks_wip = @person.tasks.where(date_complete: nil).count
+    @tasks_done = @tasks_count - @tasks_wip
+  rescue
+    redirect_to "/person_not_found"
```

Comment:
I was planning to eventually take the text from the url and turn it into a custom error message, so this particular 404 redirect would have said at the top of the page: "Person not found. Please check your number and try your call again."

Anyway, I will definitely be checking out some dynamic 404 tutorials like you've recommended! In the mean time, I've left my own custom error page, because--- well, basically, I didn't want to mess with the styles on the static 404 page just now. >_>